src/react-components/layout/Header.js

```diff
@@ -50,7 +50,7 @@ export function Header() {
             </IfFeature>
             <IfFeature name="show_community_link">
               <li>
-                <a href={configs.link("community", "https://discord.gg/wHmY4nd")}>
+                <a href={configs.link("community", "https://discord.gg/dFJncWwHun")}>
```

Comment:
This might be a good opportunity to just remove this URL here, and also in settings-menu.js below, since it is already configurable (via `configs.link()`). I believe just removing the second parameter should be sufficient.

Not a big deal though, I'd be okay with pushing this PR either way.